tests/model_inheritance/models.py

```diff
@@ -186,3 +186,7 @@ class Child(Parent):
 
 class GrandChild(Child):
     pass
+
+
+class ItalianRestaurantManyParents(ItalianRestaurant, Place):
+    place_ptr_2 = models.OneToOneField(Place, models.CASCADE, parent_link=True)
```

Comment:
```suggestion
class ItalianRestaurantManyParents(ItalianRestaurant, Place):
    pass
```

Reply:
I am aware of the actual regression, but just wanted to check that if the field has an attribute attname, it passes or not, the actual registration was there in my initial commits you can check.